spec.html

```diff
@@ -40313,9 +40326,11 @@ <h1>Promise.race ( _iterable_ )</h1>
         <emu-alg>
           1. Let _C_ be the *this* value.
           1. Let _promiseCapability_ be ? NewPromiseCapability(_C_).
+          1. Let _promiseResolve_ be GetPromiseResolve(_C_).
+          1. IfAbruptRejectPromise(_promiseResolve_, _promiseCapability_).
           1. Let _iteratorRecord_ be GetIterator(_iterable_).
           1. IfAbruptRejectPromise(_iteratorRecord_, _promiseCapability_).
-          1. Let _result_ be PerformPromiseRace(_iteratorRecord_, _C_, _promiseCapability_).
+          1. Let _result_ be PerformPromiseRace(_iteratorRecord_, _C_, _promiseCapability_, _promiseResolve_).
```

Comment:
PerformPromiseRace still only takes 3 arguments

Reply:
Ah... sorry. I think that was a bad rebase fixed.